modules/analysis/src/error.rs

```diff
@@ -0,0 +1,67 @@
+use actix_http::StatusCode;
+use actix_web::body::BoxBody;
+use actix_web::{HttpResponse, ResponseError};
+use sea_orm::DbErr;
+use trustify_common::error::ErrorInformation;
+use trustify_common::id::IdError;
+use trustify_common::purl::PurlErr;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
If this is a copy of the same errors from Fundamental, perhaps we can share?

If sharing seems ill-advised, do we need all of these variants? (IdKey for instance?)

Reply:
related: I think it'd be fine to have a dependency upon `fundamental`, particularly if we adjust the output of analysis to use fundamental things like `PurlHead` or whatnot, to throw in UUIDs.

Reply:
opted to amend and keep for the time being - it is unclear if modules/analysis could be used standalone someday ... hoping that is ok.